src/Microsoft.Health.Fhir.Core/Features/Operations/Export/Models/ExportJobRecord.cs

```diff
@@ -40,7 +40,8 @@ public class ExportJobRecord : JobRecord, IJobData
             string storageAccountContainerName = null,
             int parallel = 0,
             int schemaVersion = 2,
-            int typeId = (int)JobType.ExportOrchestrator)
+            int typeId = (int)JobType.ExportOrchestrator,
+            bool smart = false)
```

Comment:
Should we user a more meaningful name instead of "smart" only? Maybe "isSmartRequest" or "SmartRequest"?

Reply:
Updated.